app/models/user.rb

```diff
@@ -356,6 +373,19 @@ def send_digest_email
     end
   end
 
+  def send_digest_email_spam
```

Comment:
Could you please add tests for these model methods?
Everything else looks good...great job :tada:

Reply:
Sure, I will add it soon. Thanks!!